### PR TITLE
Fix broken links including multiple dashes

### PR DESCRIPTION
**Pull request by SIkebe**
Statiq.Framework removes multiple dashes when optimizing file name.
https://github.com/statiqdev/Statiq.Framework/issues/114

I fixed some links broken by this behavior.

### CODE REVIEW

File: input/blog/posts/net-foundation-february-2019-update.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This month's newsletter includes:</p>
 
 <ul>
 <li><u><a href="/blog/2019/01/14/why-you-should-join-part-1-join-as-a-voting-member">Why You Should Join: Part 1 - Voting Member</a></u></li>
-<li><a href="/blog/2019/01/16/why-you-should-join-part-2---corporate-sponsor">Why You Should Join: Part 2 - Corporate Sponsor</a></li>
+<li><a href="/blog/2019/01/16/why-you-should-join-part-2-corporate-sponsor">Why You Should Join: Part 2 - Corporate Sponsor</a></li>
```
Comment (SIkebe) [inline input/blog/posts/net-foundation-february-2019-update.md:26]: Should I rename the original markdown file, too?

**Comment by SIkebe** [comment]
Reduced some link validation failures. There are still left a lot.

image
↓
image

File: input/blog/posts/net-foundation-february-2019-update.md (hunk 1 above)
Comment (clairernovotny) [reply input/blog/posts/net-foundation-february-2019-update.md:26]: Thanks. Yes, can you please rename the original file and then we can merge the changes?

Comment (SIkebe) [reply input/blog/posts/net-foundation-february-2019-update.md:26]: @clairernovotny I renamed the file and rebased onto current master.

**Comment by SIkebe** [comment]
I found colon is not rendered correctly while testing this PR. Should raise another issue?

image

**Pull request merged**